clidocstool.go

```diff
@@ -88,21 +88,17 @@ func fileExists(f string) bool {
 	return !info.IsDir()
 }
 
-func copyFile(src string, dest string) error {
-	srcFile, err := os.Open(src)
+func copyFile(src string, dst string) error {
+	sf, err := os.Open(src)
 	if err != nil {
 		return err
 	}
-	defer srcFile.Close()
-
-	destFile, err := os.Create(dest)
+	defer sf.Close()
+	df, err := os.OpenFile(dst, os.O_CREATE|os.O_WRONLY, 0600)
```

Comment:
Perhaps use the new formatting while we're changing this;

```suggestion
	df, err := os.OpenFile(dst, os.O_CREATE|os.O_WRONLY, 0o600)
```

Reply:
Perhaps we should use `gofumpt` more in our repositories (see https://github.com/moby/sys/pull/92)

Reply:
Good idea!

Reply:
We can add linter in a follow-up.

Reply:
https://github.com/docker/cli-docs-tool/issues/13